### ISSUE TITLE
enhance action logging

### ISSUE THREAD

**Issue by MohabMohie** [ShaftHQ/SHAFT_ENGINE#1799]
اظن اللى مكتوب فى allure محتاج يبان فى ال logs.
مع فكرة اضافة حرف الجر مابين ال action وال target element.
وياسلام لو ال element name يبان فى ال test data.
وكمان نفكر نشيل ال double quotes حوالين ال element name علشان محدش يفهم غلط ان ده اللى اتكتب.

**Comment by AhmdZanoon** [comment]
@KyrillosNageh yes

**Comment by AhmdZanoon** [comment]
@KyrillosNageh yes for type action , no for assert statnent , no issue with assert statement for now

**Comment by KyrillosNageh** [comment]
@MohabMohie  @AhmdZanoon 
Do you think this output is acceptable for the Type Action?

![image](https://github.com/user-attachments/assets/62f5eb83-26d0-4571-a515-5d7f4d5d237c)
